ebr_connector/prepacked_queries/multi_jobs.py

```diff
@@ -1,11 +1,15 @@
 """
 A collection of queries that provide multiple results as an array of dicts
 """
+
 from elasticsearch_dsl import Q, A
+from deprecated.sphinx import deprecated
 
 from ebr_connector.schema.build_results import BuildResults
 from ebr_connector.prepacked_queries.query import make_query, DETAILED_JOB, JOB_MINIMAL
 
+
+@deprecated(version="0.1.1", reason="It is planned to replace these functions with a DSL")
```

Comment:
Two things for reason:
1. Can we set this to a variable, maybe prepacked_dep_msg?
2. Since we are uncertain that we will replace with a DSL, but we know that we will remove, let's have the message read "The prepacked_queries modules will be removed. A replacement is under consideration but not guaranteed."